index.ts

```diff
@@ -5,7 +5,13 @@
 // -----------------------------------------------------------------------------
 
 import { format } from 'util';
-import clc from 'cli-color';
+import {
+  blue as coloretteBlue,
+  italic as coloretteItalic,
+  red as coloretteRed,
+  underline as coloretteUnderline,
+  yellow as coloretteYellow
```

Comment:
```suggestion
  yellow as coloretteYellow,
```

A trailing comma here is preferable, apparently.

Reply:
Added it